src/charm.py

```diff
@@ -24,8 +24,8 @@ def __init__(self, *args):
 
         super().__init__(*args)
 
-        self.stored.set_default(alertmanagers=[])
```

Comment:
Aren't these the same changes in https://github.com/canonical/prometheus-operator/pull/12 . If so, I think we should we just merge that pull request and make this pull request only about changing the Grafana interface name. I don't see the point of duplicating that work.